src/Globals.h

```diff
@@ -17,6 +18,13 @@ enum class RoverState {
 	LookingForTennisball,
 };
 
+namespace control {
+enum class DriveMode;
+class SwerveController;
+} // namespace control
+
+using control::DriveMode;
```

Comment:
Since we include the header file, we don't need the forward declaration. Also the other stuff is unused?